### DISCUSSION TITLE
UPAC Colloquia

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#182] [Astrophysics and Physics]
"During the Utrecht Philosophy of Astronomy & Cosmology colloquium reading session next Tuesday [16:00-17:00 CE(S)T], we will be discussing a paper by Milan M. Ćirković and Slobodan Perović: "Alternative explanations of the cosmic microwave background: A historical and an epistemological perspective"."  (Online via Teams, or in person at Utrecht University.)

We historically trace various non-conventional explanations for the origin of the cosmic microwave background and discuss their merit, while analyzing the dynamics of their rejection, as well as the relevant physical and methodological reasons for it. It turns out that there have been many such unorthodox interpretations; not only those developed in the context of theories rejecting the relativistic (“Big Bang”) paradigm entirely (e.g., by Alfvén, Hoyle and Narlikar) but also those coming from the camp of original thinkers firmly entrenched in the relativistic milieu (e.g., by Rees, Ellis, Rowan-Robinson, Layzer and Hively). In fact, the orthodox interpretation has only incrementally won out against the alternatives over the course of the three decades of its multi-stage development. While on the whole, none of the alternatives to the hot Big Bang scenario is persuasive today, we discuss the epistemic ramifications of establishing orthodoxy and eliminating alternatives in science, an issue recently discussed by philosophers and historians of science for other areas of physics. Finally, we single out some plausible and possibly fruitful ideas offered by the alternatives.

See "Winter Semester" Schedule https://www.uu.nl/en/research/utrecht-philosophy-of-astronomy-cosmology/events/upac-colloquium
Sign up [here](https://mailman.science.uu.nl/mailman/listinfo/upac-l) for the UPAC Colloquium Mailing List.

This paper was published in 2018: https://www.sciencedirect.com/science/article/abs/pii/S1355219816302039

**Comment by ExpEarth** [comment]
Thanks Louis.  Does the "online in Teams" mean we can listen in?  They are interesting guys and they have a book coming out soon on the same topic.

In case the link to the paper does not work for some, I had added the PDF to the discussion Mike started about the CMB (Disc #179)
________________________________
From: Louis Marmet ***@***.***>
Sent: November 10, 2023 10:32 AM
To: a-cosmology-group/acg ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [A Cosmology Group] Nov. 14 UPAC Colloquium - "Alternative explanations of the cosmic microwave background: A historical and an epistemological perspective" (Discussion #182)


"During the Utrecht Philosophy of Astronomy & Cosmology colloquium reading session next Tuesday [16:00-17:00 CE(S)T], we will be discussing a paper by Milan M. Ćirković and Slobodan Perović: "Alternative explanations of the cosmic microwave background: A historical and an epistemological perspective"." (Online via Teams, or in person at Utrecht University.)

We historically trace various non-conventional explanations for the origin of the cosmic microwave background and discuss their merit, while analyzing the dynamics of their rejection, as well as the relevant physical and methodological reasons for it. It turns out that there have been many such unorthodox interpretations; not only those developed in the context of theories rejecting the relativistic (“Big Bang”) paradigm entirely (e.g., by Alfvén, Hoyle and Narlikar) but also those coming from the camp of original thinkers firmly entrenched in the relativistic milieu (e.g., by Rees, Ellis, Rowan-Robinson, Layzer and Hively). In fact, the orthodox interpretation has only incrementally won out against the alternatives over the course of the three decades of its multi-stage development. While on the whole, none of the alternatives to the hot Big Bang scenario is persuasive today, we discuss the epistemic ramifications of establishing orthodoxy and eliminating alternatives in science, an issue recently discussed by philosophers and historians of science for other areas of physics. Finally, we single out some plausible and possibly fruitful ideas offered by the alternatives.

See "Winter Semester" Schedule https://www.uu.nl/en/research/utrecht-philosophy-of-astronomy-cosmology/events/upac-colloquium
Sign up here<https://mailman.science.uu.nl/mailman/listinfo/upac-l> for the UPAC Colloquium Mailing List.

This paper was published in 2018: https://www.sciencedirect.com/science/article/abs/pii/S1355219816302039

**Reply by budrap00** [reply]
The 28 Nov session also looks quite interesting. From the abstract:

`Contrary to the habitual claims of a universe, its beginnings, and its constituents revealing themselves to physicists as the result of increasingly precise empirical observations, I suggest that physical cosmology’s history is better understood as a steady unravelling, the disassembling of a total unity first achieved in the abstract with the first mathematical description of the universe in the early twentieth century. Rather than a picture of the universe slowly coming together, I contend that in physical cosmology we have a picture of a universe that is steadily falling apart while physicists struggle to hold on to the totality originally achieved. In this description, the momentum is not one of ever upward progress in physics, but one of decline. It is a universe that, I argue, always seems just about to run away from cosmologists who risk the inclusion of increasingly speculative and possibly chimeral beings in their efforts to keep the totalising unity of the universe together.`

**Reply by RedshiftDrift** [reply]
@ExpEarth when you sign up UPAC sends you the paper through their mailing list, with colloquia announcements and a [Teams link](https://support.microsoft.com/en-us/topic/what-is-microsoft-teams-3de4d369-0167-8def-b93b-0eb5286d7a29).

**Comment by ExpEarth** [comment]
Thanks, Louis!  I sent the info along to Milan Cirkovic and he was interested to hear about it.  His book will be out soon and I think he is looking for people to do book reviews on it.  It's by Cambridge University Press - am not sure the title.

**Reply by RedshiftDrift** [reply]
I might have some time to read Ćirković's book (as long as it's not in Serbian!)  His paper "Alternative explanations of the CMB..." is missing some important alternatives - I'd like to know if these omissions are addressed in his new book.

**Reply by HanDeBruijn** [reply]
For @HanDeBruijn the UPAC Colloquium in Utrecht, the Netherlands, is almost around the corner. 
Unfortunately, he is not very mobile these days 😥
Apart from that, I find the following sentence from the abstract disturbing.
>Finally, we single out some plausible and possibly fruitful ideas offered by the alternatives.

I cannot remember that ever in history there has been made a sensible prediction of future developments in science.
Important discoveries come as a surprise. They are always dependent on creativity of some individual researcher(s).

**Reply by ExpEarth** [reply]
@RedshiftDrift

That would be good.  I'll let Milan know!

**Comment by MJoseph315** [comment]
While we’re on this subject, what are all of your thoughts on the idea that dust grains can, at least partially, explain the CMBR without a big bang? This idea has been proposed and explored by scientists like Jayant Narlikar, but it has also been met with a fair degree of criticism.

**Reply by RedshiftDrift** [reply]
More recently Václav Vavryčuk has also published many papers on this topic [https://academic.oup.com/mnras/article/478/1/283/4975800](https://academic.oup.com/mnras/article/478/1/283/4975800) The "dust" model explains some features of the CMB, and Vavryčuk also claims that dust explains redshift, therefore no Big Bang.

However, dust produces absorption at shorter wavelengths (distant galaxies appear redder - not to be confused with redshift). This absorption can easily be calcuated and the answer yields much more absorption than is observed for distant galaxies.  In other words, *if there was enough dust to produce the CMB (long wavelengths), we would not be able to see the light (short wavelengths) from distant galaxies because it would all be absorbed.*

Instead of "regular dust" (e.g. sand grains) Narlikar proposed "cosmic needles" (small elongated metallic particles) and Lerner proposed a "plasma", both of which would be responsible for the CMB.  The argument against these is the same as above: *none of them can have* ***both*** *diffusion and absorption properties that explain the CMB without too much absorption in another part of the spectrum.*

**Reply by MJoseph315** [reply]
If you don't think these redshift mechanisms are sufficient to explain the redshift, what are the specifics of your redshift mechanism that you think makes it succeed where you think these other mechanisms have failed?

**Comment by bgreenman** [comment]
I think we are just beginning to test alternative theories on alternative origins of CMB.

**Reply by RedshiftDrift** [reply]
No we're not, the question about the temperature of space dates back from at least 1896[^1] and alternative models were proposed as soon as the CMB was discovered.[^2]

[^1]: C. É. Guillaume, “La Température de L’Espace,” La Nature, vol. 1214, pp. 210–211, 234 (1896).
[^2]: F. Hoyle, “On the origin of the microwave background.,” The Astrophysical Journal, vol. 196, pp. 661–670 (1975).

**Reply by HanDeBruijn** [reply]
@RedshiftDrift .
The second reference has been declared as _Essential reading_ in the section [Support by Hoyle](http://www.alternatievewiskunde.nl/cosmology/Hoyle.htm) at my website. But in retrospect, there is little substantial in it, especially if it comes to _alternative models_ concerning the CMB. The paper seems to be considered by some as beautiful science(-fiction) proze. Maybe that's all, though.

**Reply by HanDeBruijn** [reply]
@RedshiftDrift . Concerning the first reference.

[Complete and Commented Translation of Guillaume’s 1896 Paper on the Temperature of Space](https://www.ifi.unicamp.br/~assis/Am-J-Phys-V88-p1140-1144(2020).pdf), substantial, at page 9:

>We conclude that the radiation of the stars alone would maintain a test particle we suppose might have been placed at different points in the sky at a temperature of 338/60 = **5.6** absolute = −267.4 centigrade.

**Reply by bgreenman** [reply]
Guillaume's 1896 Paper on the Temperature Of Space is a good starting point for looking at alternative CMB models

**Comment by RedshiftDrift** [comment]
***[UPAC Colloquium] Speaker Session: Dark Matter - Tuesday 9 Dec. at*** **14h30 UTC**

Watch online with Micromoft Teams.

-------- Forwarded Message --------
Subject: 	[UPAC Colloquium] Speaker Session: Dark Matter
Date: 	Mon, 8 Dec 2025 07:56:59 +0000
From: 	Ferreiro de Aguiar, A.E. (Antonio) via Upac-l <upac-l@lists.science.uu.nl>
Reply-To: 	Ferreiro de Aguiar, A.E. (Antonio) <a.e.ferreirodeaguiar@uu.nl>
To: 	Martens, N.C.M. (Niels) via Upac-l <upac-l@lists.science.uu.nl>


Dear all,

 

Tomorrow, 9 December 2025, 15h30-17h00, we will have a UPAC colloquium speaker season. [Chris Smeenk](https://cjsmeenk.github.io/)  [Western University] will give a talk about: 


How Do We Measure Dark Matter? Let us Count the Ways: A Study in Theory-Mediated Measurement

This talk develops and defends a version of measurement realism, a form of scientific realism grounded in the epistemic achievements of successful measurement practices. The central idea is that when multiple independent measurement techniques achieve stability and convergence under increasing precision, this warrants realist commitment to a "thin'' concept of the measured entity: one specifying enough identifying features (related to causal roles) for stable inductive and explanatory practices, while remaining open to further elaboration and even significant theoretical revision. The case for molecular reality, as articulated in George Smith's account of theory-mediated measurement, exemplifies the strengths of this view. Precision measurements of Avogadro's number (and other molecular scale parameters) from diverse phenomena converged, despite employing distinct "linking assumptions'' related to different aspects of molecules. This convergence transformed the atomic hypothesis into accepted science even as it revealed new complexities, and these achievements survived and supported the transition to quantum theory a decade later.  This measurement-based framework promises to provide useful insights regarding contemporary dark matter research, through examining how multiple observational techniques (rotation curves, cluster dynamics, gravitational lensing, CMB acoustic peaks, BBN, and others) constrain dark matter's properties. Each method involves substantive linking assumptions connecting observations to dark matter characteristics. However, there are important disanalogies with molecular reality: significant gaps in coherence between different measurement approaches, outstanding puzzles about the nature of what is being measured, and unresolved questions about the appropriate ``thinness'' of the dark matter concept.  These complications raise questions about the scope and limits of measurement realism that I will explore alongside recent philosophical work on dark matter. I contend that measurement realism offers a distinctive and attractive alternative to existing positions in the debate over dark matter, one that philosophers have not adequately considered despite its advantages in accounting for both the successes and limitations of current evidence.

Location: Buys Ballotgebouw - Room BBG 4.01  (see below for Teams link).
For travel directions and the upcoming schedule, see: https://www.uu.nl/en/research/utrecht-philosophy-of-astronomy-cosmology/events/upac-colloquium.

_________________________________________________________________________
Microsoft Teams 
[Join the meeting now](https://teams.microsoft.com/l/meetup-join/19%3ameeting_MGEyNmMxN2UtN2QwYS00Mzg1LTlkYzAtMmNjMjZhOWUyZWE0%40thread.v2/0?context=%7b%22Tid%22%3a%22d72758a0-a446-4e0f-a0aa-4bf95a4a10e7%22%2c%22Oid%22%3a%22cd5cd402-23cc-402f-ba04-dd0489088cc6%22%7d)
Meeting ID: 388 660 062 25
Passcode: CeAiry
_________________________________________________________________________
 
Best,
Antonio Ferreiro, on behalf of the UPAC group
